### DISCUSSION TITLE
How can I put video captions?

### DISCUSSION THREAD

**Discussion by sunglee** [cloudinary-community/next-cloudinary#428] [Q&A]
Hi community,

I am new to this library and I am trying to use `CldVideoPlayer` component. In the documentation, it doesn't mention about the video captions/subtitles. Is it possible to configure in `CldVideoPlayer` component?

**Comment by colbyfayock** [comment]
@sunglee i dont believe it's current supported as the way the player initializes doesn't take advantage of the method that allows you to pass in text tracks

i created a new Issue to address this: https://github.com/cloudinary-community/next-cloudinary/issues/430

**Reply by colbyfayock** [reply]
@sunglee its been a while, but released an update that supports captions and subtitles https://next.cloudinary.dev/cldvideoplayer/configuration#text-tracks
